MyMentorProject/src/main/java/com/Deny/MyMentorProject/Services/UserService.java

```diff
@@ -0,0 +1,51 @@
+package com.Deny.MyMentorProject.Services;
+
+import com.Deny.MyMentorProject.User.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    User user;
+
+
+  public UserService(User user) {       this.user = user;
+ }
+
+
+    public String helloMessage() {
+        return "Hello My Friend !";}
```

Comment:
format your code. IntelliJ can do it for you. On Mac, CMD+OPTION+L. On Windows, CTRL+ALT+L.